Makefile

```diff
@@ -20,15 +18,19 @@ debug: # debug creates an executable with optimizations off, suitable for debugg
 
 .PHONY: bin/windows
 bin/windows: # create windows binaries
-	GOOS=linux GOARCH=amd64 go build -o ./internal/assets/ceb/ceb ./cmd/vagrant-entrypoint
-	cd internal/assets && go-bindata -pkg assets -o prod.go -tags assetsembedded ./ceb
+	@test -s "thirdparty/proto/api-common-protos/.git" || { echo "git submodules not initialized, run 'git submodule update --init --recursive' and try again"; exit 1; }#
 	GOOS=windows GOARCH=amd64 CGO_ENABLED=$(CGO_ENABLED) go build -ldflags $(GOLDFLAGS) -tags assetsembedded -o ./vagrant.exe ./cmd/vagrant
 
 .PHONY: bin/linux
 bin/linux: # create Linux binaries
 	@test -s "thirdparty/proto/api-common-protos/.git" || { echo "git submodules not initialized, run 'git submodule update --init --recursive' and try again"; exit 1; }
 	GOOS=linux GOARCH=amd64 $(MAKE) bin
 
+.PHONY: bin/linux-386
+bin/linux: # create Linux binaries
```

Comment:
Typo on the name here - overwriting the previous target

Reply:
Yes! this was a typo. Thanks and fixed